ml-unit-test-client/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 	id "com.jfrog.bintray" version "1.8.0"
 
 	// ml-gradle is used for deploying a test application so that this project itself can be tested
-	id "com.marklogic.ml-gradle" version "3.4.1"
+	id "com.marklogic.ml-gradle" version "3.6.3"
```

Comment:
ml-gradle is up to 3.7.1. any reason not to use the current version?

Reply:
I'll bump it up after merging this into dev